src/index.js

```diff
@@ -372,4 +372,20 @@ export default class KintoClient {
     return this.execute(requests.deleteBucket(_bucket, reqOptions))
       .then(res => res.json);
   }
+
+  /**
+   * Deletes all buckets on the server.
+   *
+   * @ignore
+   * @param  {Object}  options         The options object.
+   * @param  {Boolean} options.safe    The safe option.
+   * @param  {Object}  options.headers The headers object option.
+   * @return {Promise<Object, Error>}
+   */
+  @support("1.4", "2.0")
```

Comment:
Hmm would be read as well as *support for this feature is provided from version 1.4 to 2.0*...